services/invoke/Dockerfile

```diff
@@ -1,53 +1,153 @@
-FROM alpine:3.17 as xformers
-RUN apk add --no-cache aria2
-RUN aria2c -x 5 --dir / --out wheel.whl 'https://github.com/AbdBarho/stable-diffusion-webui-docker/releases/download/6.0.0/xformers-0.0.21.dev544-cp310-cp310-manylinux2014_x86_64-pytorch201.whl'
+# syntax=docker/dockerfile:1.4
```

Comment:
Please remove this since it is not part of the official OCI image format.

or is it now?

Reply:
yeah the syntax statement, but no worries, I want to dig deeper here anyway since a lot of stuff changed.